### PR TITLE
Tests cleanups

### PR DESCRIPTION
**Pull request by adrian17**
(Didn't touch cpython_bytecode tests, as they don't seem to be working anyway?)
This massively speeds up time of running all tests, to 5-10 seconds max.

### CODE REVIEW

File: tests/snippets/append.py [hunk 1]
```diff
@@ -1,2 +0,0 @@
-x = []
```
Comment (windelbouwman) [inline tests/snippets/append.py]: Why did you remove this file? I agree the test makes little sense, but still, it is a valid test.

Comment (adrian17) [reply tests/snippets/append.py]: Because other tests in `list.py` do the same job. No point testing the same thing twice.
